src/pages/homepage.tsx

```diff
@@ -96,47 +96,51 @@ const HomePage = () => {
         <Container gridCols="12" gridGutters="11" paddingX={false} gray={true}>
           <ButtonBlockList title={t('homepage:cta_public.title')}>
             <ButtonBlock
-              title={t('homepage:cta_public.bloc1.title')}
+              title={t('homepage:cta_public.case_studies.title')}
```

Comment:
In the case of these blocks, to allow the text to evolve without changing the key, I would have kept a generic name.